dub.json

```diff
@@ -37,9 +37,37 @@
         "source/scpp/build/numeric.o",
         "source/scpp/build/uint128_t.o"
     ],
+    "sourceFiles-windows": [
+        "source/scpp/build/BallotProtocol.obj",
+        "source/scpp/build/ByteSliceHasher.obj",
+        "source/scpp/build/DSCPUtils.obj",
+        "source/scpp/build/DUtils.obj",
+        "source/scpp/build/HashOfHash.obj",
+        "source/scpp/build/Hex.obj",
+        "source/scpp/build/KeyUtils.obj",
+        "source/scpp/build/LocalNode.obj",
+        "source/scpp/build/Logging.obj",
+        "source/scpp/build/Math.obj",
+        "source/scpp/build/NominationProtocol.obj",
+        "source/scpp/build/QuorumSetUtils.obj",
+        "source/scpp/build/SCP.obj",
+        "source/scpp/build/SCPDriver.obj",
+        "source/scpp/build/SHA.obj",
+        "source/scpp/build/SecretKey.obj",
+        "source/scpp/build/Slot.obj",
+        "source/scpp/build/StrKey.obj",
+        "source/scpp/build/crc16.obj",
+        "source/scpp/build/jsoncpp.obj",
+        "source/scpp/build/marshal.obj",
+        "source/scpp/build/numeric.obj",
+        "source/scpp/build/uint128_t.obj",
+        "lib/win64/sodium.lib"
+    ],
 
-    "dflags": [ "-extern-std=c++14" ],
-    "lflags": [ "-lsodium", "-lstdc++", "-lsqlite3" ],
+    "dflags-posix":   [ "-extern-std=c++14"],
+    "dflags-windows": [ "-extern-std=c++14", "-mtriple=x86_64-windows-msvc" ],
```

Comment:
Can you check if the triple is necessary ? That should be the default (e.g. on your machine, do `ldc2 --version` and you'll see a "default target")

Reply:
```
Run ${DC} --version
LDC - the LLVM D compiler (1.20.0):
  based on DMD v2.090.1 and LLVM 9.0.1
  built with LDC - the LLVM D compiler (1.20.1)
  Default target: x86_64-pc-windows-msvc
  Host CPU: skylake
```
I don't think I need it after testing it.
Default values are set according to the platform.